Dockerfile

```diff
@@ -147,10 +157,12 @@ FROM ubuntu${GPROFILER_BUILDER_UBUNTU}
 WORKDIR /app
 
 # for Aarch64 - it has no .whl file for psutil - so it's trying to build from source.
+
 RUN set -e; \
     apt-get update && \
     apt-get upgrade -y && \
     apt-get install --no-install-recommends -y python3-pip && \
+    apt-get install -y libicu66 && \
```

Comment:
Why is this one required? for dotnet-trace?

Reply:
Yes, it is required by dotnet-trace. Reference:
https://stackoverflow.com/questions/59119904/process-terminated-couldnt-find-a-valid-icu-package-installed-on-the-system-in
setting the ENV variable was not enough

Reply:
I see. For `pyi.Dockerfile`, we build the binary in the image but the image itself isn't used, only the binary. We use `staticx` to pack all necessary libraries into the binary. Need to ensure that the result binary actually gets the library & is able to profile dotnet applications.

Reply:
(if `gprofiler` binary is able to profiler dotnet apps when started in an environment where `libicu66` is not isntalled - then that settles it)